Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml

```diff
@@ -67,7 +67,7 @@ configuration:
   advanced: true
   required: false
 - display: Time field type
```

Comment:
we wanted to add here the "additional info" for the customers to know we are now taking the first custom value, no?